### PR TITLE
Fix missing changes in Weave manifest

### PR DESCRIPTION
**Pull request by hakman**
Seems that these were missing compared to latest version on the weave site.

### CODE REVIEW

File: upup/models/cloudup/resources/addons/networking.weave/k8s-1.12.yaml.template [hunk 1]
```diff
@@ -36,6 +36,14 @@ rules:
       - get
       - list
       - watch
+  - apiGroups:
```
Comment (olemarkus) [inline upup/models/cloudup/resources/addons/networking.weave/k8s-1.12.yaml.template:39]: Does weave actually fetch resources using the deprecated api groups? If not, this is not necessary. It doesn't do any harm either though.

Comment (hakman) [reply upup/models/cloudup/resources/addons/networking.weave/k8s-1.12.yaml.template:39]: Best to keep it as close as possible to their official manifest, I guess. Thanks for checking this.
https://github.com/weaveworks/weave/blob/9f00f78d3b9d5a8a31fdd90ec691095028e2690a/prog/weave-kube/weave-daemonset-k8s-1.11.yaml#L28-L35

**Pull request merged**